clients/web/src/templates/widgets/groupAdminList.jade

```diff
@@ -17,10 +17,12 @@
                 i.icon-down-big
                 i.icon-down-dir
               ul.dropdown-menu.pull-right(role="menu")
-                li(role="presentation"): a.g-demote-moderator(role="menuitem")
+                li(role="presentation")
+                  a.g-demote-moderator(role="menuitem")
                   i.icon-minus
                   | Demote to moderator
-                li(role="presentation"): a.g-demote-member(role="menuitem")
+                li(role="presentation")
+                  a.g-demote-member(role="menuitem")
```

Comment:
This is not equivalent -- the icon and text used to be children of the anchor tag, now they are siblings, so the links in the dropdown are broken.